src/body.css

```diff
@@ -0,0 +1,4 @@
+body {
```

Comment:
Avoid using tag names for styling, except for `html` and `body`. Consider using a class or id selector instead.